ext/session/session.c

```diff
@@ -2711,6 +2723,22 @@ static const zend_function_entry session_functions[] = {
 
 /* {{{ SessionHandlerInterface functions[]
 */
+static const zend_function_entry php_session_save_handler_iface_functions[] = {
+	PHP_ABSTRACT_ME(SessionSaveHandlerInterface, open, arginfo_session_class_open)
```

Comment:
@Ocramius 
About simplicity, the API is simple enough. 

Simpler is better, but not too simple. 
The proposed interface is not able to do the session management task correctly/fully. Too simple is wrong.

BTW, current session module does not manage session data by timestamp, it cannot work correctly. Session module is too simple to do the required task correctly. This has to be fixed also, but this issue is irrelevant to session save handler.